cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -127,13 +167,27 @@ func runCreateManagedJob(cmd *cobra.Command, args []string) (err error) {
 	if err != nil {
 		return err
 	}
-
+	
 	// create the job
 	job, err := createJob(client)
-
+	
 	if err != nil {
 		return err
 	}
+	
+	mc, err := isManagementCluster()
+	if err != nil{
+		logger.Warn("Could not determine if cluster is Managed Cluster")
+	}
+
+	if mc { // print dynatrace url for managed clusters only		
+		dtInstance, err := getDynatraceInstance() 
+		if err != nil {
+			logger.Info("Could not get Dynatrace instance")
+		} else {
+			defer fmt.Fprintf(cmd.OutOrStdout(), "\nCheck dynatrace logs at: %s\n", dtInstance)
```

Comment:
Why do we need a defer here?

Reply:
@bmeng  The idea was to print this message as the last line on cli